101-backup-backupinstances-report/README.md

```diff
@@ -0,0 +1,38 @@
+# Create a Logic App to send information on backup instances protected using Azure Backup via email
+
+![Azure Public Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/101-backup-backupinstances-report/PublicLastTestDate.svg)
+![Azure Public Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/101-backup-backupinstances-report/PublicDeployment.svg)
+
+![Azure US Gov Last Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/101-backup-backupinstances-report/FairfaxLastTestDate.svg)
+![Azure US Gov Last Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/101-backup-backupinstances-report/FairfaxDeployment.svg)
+
+![Best Practice Check](https://azurequickstartsservice.blob.core.windows.net/badges/101-backup-backupinstances-report/BestPracticeResult.svg)
+![Cred Scan Check](https://azurequickstartsservice.blob.core.windows.net/badges/101-backup-backupinstances-report/CredScanResult.svg)
+
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-backup-backupinstances-report%2Fazuredeploy.json)  [![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-backup-backupinstances-report%2Fazuredeploy.json)
```

Comment:
remove the first visualize button (you have 2) from all samples

Reply:
I did not understand this comment @bmoore-msft . The file is similar to checked-in https://github.com/Azure/azure-quickstart-templates/pull/8721. 

Can you please pin point the change required?